### PR TITLE
Include aws-cli and git in Dockerfile #2982

### PR DESCRIPTION
**Pull request by cicharka**
* continuation of aks/eks modules ticket

Signed-off-by: cicharka <arkadiusz.cichon@outlook.com>

### CODE REVIEW

File: .devcontainer/Dockerfile [hunk 1]
```diff
@@ -38,7 +39,13 @@ RUN : INSTALL HELM BINARY \
  && curl -fsSLO https://releases.hashicorp.com/terraform/${TERRAFORM_VERSION}/terraform_${TERRAFORM_VERSION}_linux_amd64.zip \
  && unzip terraform_${TERRAFORM_VERSION}_linux_amd64.zip -d /usr/local/bin \
  && rm terraform_${TERRAFORM_VERSION}_linux_amd64.zip \
- && terraform version
+ && terraform version \
+ && : INSTALL AWS CLI BINARY \
+ && curl -fsSLO https://awscli.amazonaws.com/awscli-exe-linux-x86_64-${AWS_CLI_VERSION}.zip \
+ && unzip awscli-exe-linux-x86_64-${AWS_CLI_VERSION}.zip \
+ && ./aws/install -i /usr/local/aws-cli -b /usr/local/bin \
+ && rm -rf awscliv2.zip ./aws \
```
Comment (to-bar) [inline .devcontainer/Dockerfile]:
```suggestion
 && rm -rf awscli-exe-linux-x86_64-${AWS_CLI_VERSION}.zip ./aws \
```

File: Dockerfile [hunk 2]
```diff
@@ -32,6 +33,13 @@ RUN : INSTALL APT REQUIREMENTS \
     && unzip terraform_${TERRAFORM_VERSION}_linux_amd64.zip -d /usr/local/bin \
     && rm terraform_${TERRAFORM_VERSION}_linux_amd64.zip \
     && terraform version \
+\
+    && : INSTALL AWS CLI BINARY \
+    && curl -fsSLO https://awscli.amazonaws.com/awscli-exe-linux-x86_64-${AWS_CLI_VERSION}.zip \
+    && unzip awscli-exe-linux-x86_64-${AWS_CLI_VERSION}.zip \
+    && ./aws/install -i /usr/local/aws-cli -b /usr/local/bin \
+    && rm -rf awscliv2.zip ./aws \
```
Comment (to-bar) [inline Dockerfile]:
```suggestion
    && rm -rf awscli-exe-linux-x86_64-${AWS_CLI_VERSION}.zip ./aws \
```

**Review by to-bar — DISMISSED** [review]

**Review by seriva — DISMISSED** [review]

File: docs/home/COMPONENTS.md [hunk 3]
```diff
@@ -43,8 +43,10 @@ Note that versions are default versions and can be changed in certain cases thro
...
-| Terraform AWS provider    | 3.71.0  | https://github.com/terraform-providers/terraform-provider-aws | [Mozilla Public License 2.0](https://github.com/terraform-providers/terraform-provider-aws/blob/master/LICENSE) |
-| Crane                     | 0.11.0  | https://github.com/google/go-containerregistry/tree/main/cmd/crane     | [Apache License 2.0](https://github.com/google/go-containerregistry/blob/main/LICENSE) |
+| Terraform AWS provider     | 3.71.0  | https://github.com/terraform-providers/terraform-provider-aws | [Mozilla Public License 2.0](https://github.com/terraform-providers/terraform-provider-aws/blob/master/LICENSE) |
+| Crane                      | 0.11.0  | https://github.com/google/go-containerregistry/tree/main/cmd/crane     | [Apache License 2.0](https://github.com/google/go-containerregistry/blob/main/LICENSE) |
+| Git                        | 2.30.2  | https://github.com/git/git | [GNU GENERAL PUBLIC LICENSE Version 2](https://github.com/git/git/blob/master/COPYING) |
```
Comment (przemyslavic) [inline docs/home/COMPONENTS.md]: We don't use a fixed version of git, so I guess this version may change over time.

**Review by przemyslavic — APPROVED** [review]

**Review by to-bar — APPROVED** [review]

**Review by seriva — APPROVED** [review]

**Pull request merged**